index.css

```diff
@@ -85,14 +101,15 @@ a {
 }
 
 .delbtn {
-    color: red;
-    background-color:white;
-    float:right;
```

Comment:
Without float:right all the delete buttons line up right next to the billNames. I think I prefer the smaller delete button and having it on the right side.
![Uploading delete button location.png…]()

Reply:
Okay, honestly I'm not so sure what should I do now.
I just changed this one and pushed it again.
I'm going for the other 2 later.